src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -495,6 +524,27 @@ public CodeSignIdentity get() {
     return stepsBuilder.build();
   }
 
+  private void appendCopyHeadersStep(ImmutableList.Builder<Step> stepsBuilder) {
+    Preconditions.checkArgument(isFrameworkBundle());
+    Preconditions.checkArgument(headers.isPresent());
+
+    Path headersFolder = bundleRoot.resolve(this.destinations.getHeadersPath());
+    stepsBuilder.add(
+        new MakeCleanDirectoryStep(
+            getProjectFilesystem(),
+            headersFolder));
+
+    for (SourcePath header : headers.get().getLinks().values()) {
```

Comment:
This copies each header individually, and doens't work correctly if the symlinktree has subfolder. Is there an easy way to copy the contents of a Symlinktree?